Cargo.toml

```
@@ -246,6 +245,7 @@ members = [
 	"utils/frame/rpc/client",
 	"utils/prometheus",
 	"utils/wasm-builder",
+	"utils/merkle-tree",
```

for clarity the path could be named same as crate:
```suggestion
	"utils/binary-merkle-tree",
```